CuratedContent/TuringWay.md

```diff
@@ -0,0 +1,34 @@
+# Reproducible and ethical science. 
+<!--deck text start-->
+An on line handbook to reproducible, ethical, and collaborative computational and data science.
+<!--deck text end-->
+
+#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+#### Publication date: June 22, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html)
+Authors | The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. 
+Length | Multi-Chapter, on line handbook
+Focus | Reproducible, ethical and collaborative data science
+
+This is a wonderful handbook to reproducible, ethical, and collaborative computational and data science.
+It is an online collaborative project that brings together the expertise and experiences of various contributors in the field of data science and research.
+It contains lovely illustrations and is written in a consistent, approachable style.
+Although it does not have an HPC focus and some of the details are specific to biomedical applications, the chapters on Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research are largely applicable to HPC/CSE.
+
+The book takes its name from Alan Turing, a pioneering mathematician and computer scientist, and aims to embody his values of openness, inclusivity, and rigor. It provides practical advice and best practices for conducting data science projects in a reproducible manner, emphasizing the importance of transparency, openness, and accessibility.
```

Comment:
This reads like ChatGPT-generated bullshit in the sense that the authors don't use these terms to explain the name. I would guess "pioneering" (a colonialism that at least some of the Turing Way contributors would not select) comes from this bio https://www.turing.ac.uk/about-us. The subsequent paragraphs are wordy and repetitive compared to just quoting the preface of the book, and don't have the copyright catastrophe of ChatGPT. Could we please use quote this instead?

> The Turing Way project is open source, open collaboration, and community-driven. We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone. Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
>
> All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use The Turing Way to understand their roles and responsibility of reproducibility in data science. You can inspect our resources on [GitHub](https://github.com/alan-turing-institute/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/alan-turing-institute/the-turing-way/blob/main/LICENSE.md)).

Reply:
Alternate quote from their GitHub readme:

> Started in 2019 as a lightly opinionated guide to data science, The Turing Way has since expanded into a series of guides on Reproducible Research, Project Design, Communication, Collaboration and Ethical Research. Each guide offers chapters on a range of topics covering best practices, guidance and recommendations. These chapters have been co-authored by contributors who are students, researchers, educators, community leaders, policy-makers and professionals from diverse backgrounds, lived experiences and domain knowledge.
>
> Our moonshot goal is to make reproducibility "too easy not to do".

Reply:
I used the first but kept the moonshot remark too.